### DISCUSSION TITLE
Points of order

### DISCUSSION THREAD

**Discussion by 111phil** [OpenSlides/OpenSlides#6366] [Ideas]
It is great that OpenSlides eventually implemented a point of order function ("GO-Antrag" in German). However, it is unfortunately not yet possible to create a _vote_ on this point of order right away from it. This would be very helpful though. Instead, one must work around by creating a new motion and vote on that.

**Comment by ostcar** [comment]
We are adding a poll for agenda items. This means, it will be possible to add a poll with a variable content and variable options to a agenda item. This is already implemented on server side and will be added to the client before OpenSlides 4.0. Maybe this was what you mean?

**Reply by 111phil** [reply]
Well, not really: Points of order must regularly be voted on. It would be great if that was easily possible, without having to create an new agenda point.

**Reply by jsangmeister** [reply]
You don't have to create a new agenda point, you can just use the current one and create a poll there. Does this suffice for you? Or what would you wish specifically?
